docs/user-guide/gateway.md

```diff
@@ -0,0 +1,93 @@
+# Gateway
+
+## Gateway Connections
+
+TODO: Write me
```

Comment:
[Fancy box](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title)?

```suggestion
!!! abstract "TODO"

    Write me
```